src/commands/cmd_bulk_insert.c

```diff
@@ -103,7 +103,7 @@ int MGraph_BulkInsert(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 	CommandCtx *context;
 	// Bulk commands should always modify slaves.
 	bool is_replicated = false;
-	context = CommandCtx_New(ctx, NULL, NULL, NULL, argv, argc, is_replicated);
+	context = CommandCtx_New(ctx, NULL, NULL, NULL, NULL, argv, argc, is_replicated);
```

Comment:
we should be replicating bulk command, is this indeed the case?

Reply:
the bulk insert is executed from the main thread, so yes.